test/helpers/webauthn_helpers.rb

```diff
@@ -0,0 +1,45 @@
+module WebauthnHelpers
+  def self.create_result(client:, challenge: nil)
+    rp_id = URI.parse(client.origin).host
+    challenge =
+      if challenge
+        Base64.urlsafe_decode64(challenge)
+      else
+        SecureRandom.random_bytes(32)
+      end
+
+    result = client.create(
+      challenge: challenge,
+      rp_id: rp_id
+    )
+
+    result["rawId"] = Base64.urlsafe_encode64(result["rawId"])
+    result["response"]["attestationObject"] =
+      Base64.urlsafe_encode64(result["response"]["attestationObject"])
+    result["response"]["clientDataJSON"] =
+      Base64.urlsafe_encode64(result["response"]["clientDataJSON"])
+    result
+  end
+
+  def self.create(webauthn_credential:, client:)
```

Comment:
I think it could be hard to understand what `WebauthnHelpers.create` will do: maybe changing this to `create_credential` could help?